From foo@baz Mon Jan  5 12:37:19 PST 2009
Date: Mon, 05 Jan 2009 12:37:19 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: agnx: fix build errors due to ssid removal

The wireless core has removed the ssid variables, so also remove
them from the agnx driver so that it builds properly.

Cc: Li YanBo <dreamfly281@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/agnx/agnx.h |    2 --
 drivers/staging/agnx/pci.c  |    6 ------
 2 files changed, 8 deletions(-)

--- a/drivers/staging/agnx/agnx.h
+++ b/drivers/staging/agnx/agnx.h
@@ -115,8 +115,6 @@ struct agnx_priv {
 	int mode;
 	int channel;
 	u8 bssid[ETH_ALEN];
-	u8 ssid[32];
-	size_t ssid_len;
 
 	u8 mac_addr[ETH_ALEN];
 	u8 revid;
--- a/drivers/staging/agnx/pci.c
+++ b/drivers/staging/agnx/pci.c
@@ -324,12 +324,6 @@ static int agnx_config_interface(struct 
 		sta_power_init(priv, BSSID_STAID);
 		agnx_write32(ctl, AGNX_BM_MTSM, 0xff & ~0x1);
 	}
-	if (conf->ssid_len != priv->ssid_len ||
-	    memcmp(conf->ssid, priv->ssid, conf->ssid_len)) {
-		agnx_set_ssid(priv, conf->ssid, conf->ssid_len);
-		priv->ssid_len = conf->ssid_len;
-		memcpy(priv->ssid, conf->ssid, conf->ssid_len);
-	}
 	spin_unlock(&priv->lock);
 	return 0;
 } /* agnx_config_interface */
